src/Maestro/tests/Scenarios/repo-policies.ps1

```diff
@@ -36,7 +36,7 @@ try {
     
     Write-Host ""
     Write-Host "Setting repository merge policy to standard"
-    Darc-Set-Repository-Policies $repoUri $branchName "--standard-automerge"
+    Darc-Set-Repository-Policies -repo $repoUri -branch $branchName -policyParams @( "--standard-automerge" )
```

Comment:
This test failed with:

https://github.com/maestro-auth-test/maestro-test1 @ 104674302 - Merge Policies: []
Repository merge policies should be standard, but were not

So the policy params are not getting interpreted correctly.